azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageNetworkInterface.java

```diff
@@ -25,7 +25,7 @@
 /**
  * Azure Network sample for managing network interfaces -
  *  - Create a virtual machine with multiple network interfaces
- *  - Configure multiple network interfaces
+ *  - Configure a network interface
```

Comment:
btw - javaDocs supports HTML, so if the intent is to show each line as an actual separate line, then there should be <p> between them, otherwise all the lines just get merged together. It;'s easy to see (at least in Eclipse) what the Javadoc will look like in the tooltip - just hover over the API.